n4m-vimeo/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This is the `package.json`. All node apps have this. Since the node app requires `dotenv`, `max-api`, and `vimeo`, that's what is installed here. You can either manually enter npm modules under `dependencies` or in the terminal, you can do `npm install dotenv` which will automatically update the package.json file.

When you run `npm install`, it checks this file and then will automatically update `package-lock.json`.